src/Workspaces/Core/Portable/Remote/RemoteServiceName.cs

```diff
@@ -22,9 +22,9 @@ namespace Microsoft.CodeAnalysis.Remote
         internal const string Prefix = "roslyn";
         internal const string IntelliCodeServiceName = "pythia";
         internal const string RazorServiceName = "razorLanguageService";
-        internal const string UnitTestingAnalysisServiceName = "unitTestAnalysis";
-        internal const string LiveUnitTestingServiceName = "liveUnitTestIncrementalBuild";
-        internal const string TestSourceLocationLookupServiceName = "unitTestSourceLookup";
+        internal const string UnitTestingAnalysisServiceName = "UnitTestingAnalysis";
+        internal const string LiveUnitTestingServiceName = "UnitTestingSourceLookup";
+        internal const string TestSourceLocationLookupServiceName = "LiveUnitTestingBuild";
```

Comment:
The names for the LUT and SourceLocation services are mixed up :)

Also consider renaming the consts as well as the enum members to match the strings

Reply:
😥Ah I didn't notice that.